src/Microsoft.ML.TensorFlow/doc.xml

```diff
@@ -4,7 +4,7 @@
 
     <member name="TensorflowTransform">
       <summary>
-        Extracts hidden layers' values from a pre-trained Tensorflow model.
+        Extracts hidden layers&apos; values from a pre-trained Tensorflow model.
```

Comment:
>&apos; [](start = 30, length = 6)

is it even necessary?

Reply:
XML requires to escap: ', ", <, >, & characters. 

To my surprise, this was displaying fine, but i have had trouble with the rest of them. Changed it for consistency. 

---
In reply to: [216144446](https://github.com/dotnet/machinelearning/pull/867#discussion_r216144446) [](ancestors = 216144446)